examples/compartment_precNMC.py

```diff
@@ -101,15 +101,21 @@ def main(argv):
 
         Cuds2dot(wrapper).render()
 
+        sizeDistribution = wet_synthesis.SizeDistribution()
+        wrapper.add(sizeDistribution)
         compartmentNetwork = wet_synthesis.CompartmentNetwork()
         wrapper.add(compartmentNetwork)
 
+        pretty_print(wrapper.get(oclass=wet_synthesis.SizeDistribution)[0])

```

Comment:
This does not match the diagram in the documentation. Up to you to solve (please not in this Pull Request).

Reply:
You're right, I'll create a new PR and modify the diagram